cli/admin/src/test/java/io/pravega/cli/admin/AbstractAdminCommandTest.java

```diff
@@ -22,28 +23,34 @@
 public abstract class AbstractAdminCommandTest {
 
     // Setup utility.
-    protected static final SetupUtils SETUP_UTILS = new SetupUtils();
-    protected static final AtomicReference<AdminCommandState> STATE = new AtomicReference<>();
+    protected static final SecureSetupUtils SETUP_UTILS = new SecureSetupUtils();
```

Comment:
Same here. Do not make this thing static.

Reply:
Made it non static.